libnavui-util/src/main/java/com/mapbox/navigation/ui/utils/internal/SvgUtil.kt

```diff
@@ -8,62 +8,65 @@ import java.io.ByteArrayInputStream
 
 object SvgUtil {
 
-    private const val CSS_RULES =
-        "text { font-family: Arial, Helvetica, sans-serif; font-size: 0.8em}"
-
     @JvmStatic
-    fun renderAsBitmapWithHeight(stream: ByteArrayInputStream, desiredHeight: Int): Bitmap? {
-        try {
+    fun renderAsBitmapWithHeight(
+        stream: ByteArrayInputStream,
+        desiredHeight: Int,
+        cssStyles: String? = null
+    ): Bitmap? {
+        return try {
             val svg = SVG.getFromInputStream(stream)
 
             svg.setDocumentWidth("100%")
             svg.setDocumentHeight("100%")
-            return if (svg.documentViewBox == null) {
+            if (svg.documentViewBox == null) {
                 null
             } else {
                 val aspectRatio = svg.documentViewBox.bottom / svg.documentViewBox.right
                 val calculatedWidth = (desiredHeight / aspectRatio).toInt()
-
-                val signboard = Bitmap.createBitmap(
-                    calculatedWidth,
-                    desiredHeight,
-                    Bitmap.Config.ARGB_8888
-                )
-                val signboardCanvas = Canvas(signboard)
-                svg.renderToCanvas(signboardCanvas)
-                signboard
+                renderBitmap(svg, calculatedWidth, desiredHeight, cssStyles)
             }
         } catch (exception: Exception) {
-            return null
+            null
```

Comment:
need log

Reply:
I don't think we should add Log statements here and we are also removing Timber. Logging is fine in debug builds. This null is already returned in the form of an object later to the caller of the API.
cc @RingerJK

Reply:
I still think we should log this, at least with `debug` logging level, it's helping us and external devs better react to problems. I think for now we cannot inject logger here, as it's singleton without constructor. Could you leave note like
```
// TODO need log
```